firetongue/FireTongue.hx

```diff
@@ -107,101 +136,123 @@ class FireTongue
 	 * @param	getDirectoryContents (optional) custom function to list the contents of a directory
 	 * @param	forceCase (optional) what case to force for flags in CSV/TSV files, and in the get() function -- default is to force uppercase.
 	 */
-	public function new(?framework:Framework, ?checkFile:String->Bool, ?getText:String->String, ?getDirectoryContents:String->Array<String>, ?forceCase:Case=Case.Upper) 
+	public function new(?framework:Framework, ?checkFile:String->Bool, ?getText:String->String, ?getDirectoryContents:String->Array<String>,
```

Comment:
It would be nice if this function could be reworked to use another parameter object, including a `customFramework` class instead of individual functions, but that should probably be postponed to the next point where a major version is deemed necessary.